library/lib/node-utils.js

```diff
@@ -1,35 +1,11 @@
-const { SourceMapConsumer } = require('source-map')
 const path = require('path')
 const childProcess = require('child_process')
 
 module.exports = {
-  evalWithSourceMap,
-  withSourceMappedError,
   evalInFork,
 }
 
-function evalWithSourceMap(source, createMap) {
-  return withSourceMappedError(createMap, () => {
-    const module = { exports: {} }
-    eval(source) // eslint-disable-line no-eval
-    return module.exports.default || module.exports
-  }, { evalOnly: true })
-}
-
-function withSourceMappedError(createMap, fn, options) {
-  return withRawErrorStack(() => {
```

Comment:
Betekend nu dat die error `Error: TypeError: x.stack.trim is not a function` ook verholpen is?

Reply:
🥳